pkg/credentials/credentials.go

```diff
@@ -254,6 +254,17 @@ func AppendCloudProviderVolumes(dpa *oadpv1alpha1.DataProtectionApplication, ds
 
 // add plugin specific specs to velero deployment
 func AppendPluginSpecificSpecs(dpa *oadpv1alpha1.DataProtectionApplication, veleroDeployment *appsv1.Deployment, veleroContainer *corev1.Container, providerNeedsDefaultCreds map[string]bool, hasCloudStorage bool) error {
+
+	init_container_resources := corev1.ResourceRequirements{}
+	if len(veleroDeployment.Spec.Template.Spec.Containers) > 0 {
+		for _, container := range veleroDeployment.Spec.Template.Spec.Containers {
+			if container.Name == "velero" {
+				init_container_resources = veleroDeployment.Spec.Template.Spec.Containers[0].Resources
+				break
+			}
+		}
+	}
```

Comment:
length check are not necessary for _for_ loops.
```suggestion
	for _, container := range veleroDeployment.Spec.Template.Spec.Containers {
		if container.Name == "velero" {
			init_container_resources = container.Resources
			break
		}
	}
```

Reply:
OMG, I forgot to change this as well. @kaovilai PR updated!